### PR TITLE
Cymetric

### PR DESCRIPTION
**Pull request by scopatz**
New cymetric recipe

### CODE REVIEW

**Comment by scopatz** [comment]
Ok, this is finally working @conda-forge/core

File: recipes/cymetric/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  url: https://github.com/{{ org }}/cymetric/archive/{{ branch }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or py34]
+
+build:
+  number: 0
+  script: python setup.py install --prefix ${PREFIX}
```
Comment (mwcraig) [inline recipes/cymetric/meta.yaml]: should this also have `--single-version-externally-managed --record record.txt`?

Comment (scopatz) [inline recipes/cymetric/meta.yaml]: Nope, it doesn't use setuptools and the setup.py does non-standard things with the prefix.

**Comment by scopatz** [comment]
Thanks!

File: recipes/cymetric/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  version: {{ version }}
+
+source:
+  fn: cymetric-{{ version }}-{{ sha256 }}.tar.gz
+  url: https://github.com/{{ org }}/cymetric/archive/{{ branch }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or py34]
```
Comment (jakirkham) [inline recipes/cymetric/meta.yaml]: Somehow we got 2 build sections. One here and one below. Even took me a while to spot it despite looking for an issue. 😞

Comment (jakirkham) [inline recipes/cymetric/meta.yaml]: Fixing in PR ( https://github.com/conda-forge/cymetric-feedstock/pull/3 ).

Comment (scopatz) [inline recipes/cymetric/meta.yaml]: OMG wow!  There is a thing to lint :)

**Pull request merged**